src/ref.ts

```
@@ -81,19 +81,13 @@
-        if (isReactElement(node)) {
-          return (node as any).ref;
-        }
-        return null;
-      };
+) => React.Ref<T> | null = node => {
+  if (node && isReactElement(node)) {
+    const ele = node as any;
+
+    // Source from:
+    // https://github.com/mui/material-ui/blob/master/packages/mui-utils/src/getReactNodeRef/getReactNodeRef.ts
+    return ele.props.propertyIsEnumerable('ref') ? ele.props.ref : ele.ref;
```

Biome (1.9.4)

[error] Do not access Object.prototype method 'propertyIsEnumerable' from target object.



(lint/suspicious/noPrototypeBuiltins)